KsApi/ServiceType.swift

```diff
@@ -526,12 +526,7 @@ extension ServiceType {
     headers["X-KICKSTARTER-CLIENT"] = self.serverConfig.apiClientAuth.clientId
     headers["Kickstarter-iOS-App-UUID"] = self.deviceIdentifier
 
-    return headers.withAllValuesFrom(self.pxHeaders)
-  }
-
-  // PerimeterX authorization header
-  internal var pxHeaders: [String: String] {
-    return self.perimeterXClient.headers()
+    return headers
```

Comment:
We're no longer setting the headers manually, SDK handles that [automatically](https://docs.perimeterx.com/docs/ios-manual-integration-v3). "Understanding the Flow"